config/helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler
-version: 0.15.1
+version: 0.15.2
```

Comment:
can you move this back to `0.15.1`? Our release process will automatically increment the chart version to batch any chart updates into a single version update.

Reply:
Done.